src/sagas/wallet.js

```diff
@@ -205,11 +218,6 @@ export function* startWallet(action) {
     }
   }
 
-  walletUtil.storePasswordHash(pin);
```

Comment:
You are storing the password somewhere else now, right?

Reply:
Yes, it is saved in the accessData, on the storage.
So it will be created where we create the accessData, which is the start method of the facade.